src/cmd/sql_cmd.h

```diff
@@ -79,6 +79,25 @@ ::openmldb::sdk::SQLClusterRouter* sr = nullptr;
 using VariableMap = std::map<std::string, std::string>;
 VariableMap session_variables = {VariableMap::value_type("execute_mode", "online")};
 
+bool isOnlineMode() {
+    for (auto& pair : session_variables) {
```

Comment:
session_variables must have the key `execute_mode`, so you can use map.find("execute_mode"), no need to loop

Reply:
Okay.